### ISSUE TITLE
Feature Proposal: 2 or more AO's 

### ISSUE THREAD

**Issue by Your401kPlan** [Kexanone/COE2_AR#1]
One of the cool features of DRO in A3 is that you could set some additional objectives on other points on the map, meaning you had to potentially be ready to do a long haul or some OSP. It would be cool to be able to set at least 2 AO locations on the map that you have to move to and complete.

**Comment by bdux** [comment]
I second that. It would be really nice to "chain" a couple AOs and create a mission flow. The restricition of the deployment point having the radius for spawning stuff would also make a lot more sense in that context.

**Comment by 0xmex** [comment]
I like the idea, but would love to have it as an optional setting. Maybe have the second AO tie on an obtainable Intel which needs to be found in order to find (as in generate) the second AO.
Maybe generate the markers first and only populate the AO once the first got completed and successfully exfiled.

**Comment by bdux** [comment]
Yes, absolutely. I would envision it as "add Objective/AO" as a menu point when you already have started and AO. This way you could simply complete the AO you started or, before deployment add as many AOs as you want.

**Comment by Kexanone** [comment]
Resolved in `2.3.0`
